indexer/packages/postgres/src/stores/funding-index-updates-table.ts

```diff
@@ -193,9 +193,17 @@ export async function findFundingIndexMap(
       options,
     );
 
+  // Assuming block time of 1 second, this should be 4 hours of blocks
+  const FOUR_HOUR_OF_BLOCKS = Big(3600).times(4);
   const fundingIndexUpdates: FundingIndexUpdatesFromDatabase[] = await baseQuery
     .distinctOn(FundingIndexUpdatesColumns.perpetualId)
     .where(FundingIndexUpdatesColumns.effectiveAtHeight, '<=', effectiveBeforeOrAtHeight)
+    // Optimization to reduce number of rows needed to scan
```

Comment:
are we guaranteed to have all the necessary funding index updates in a 4 hour window?

Reply:
turns out these are updated every hour so we have 4 hours to create a little buffer in case block times improve significantly.